### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by ncammarata** [facebookresearch/DrQA#51]
Fantastic work so far! I'm curious if you have done any work introducing transfer learning DrQA in a similar style to how this has been done so well for ImageNet? 

I could see many use-cases of slightly customizing Wikipedia with a smaller corpus (a law firm's documents, medical dataset, etc) that aren't big enough by themselves to learn from.

**Comment by ajfisch** [comment]
Hi @ncammarata, in the DrQA paper we do experiment with multitasking on a variety of question domains. In general we have found that the document reader is fairly transferable, depending on the target domain.

If the target domain is very different from SQuAD/Wikipedia (the main training source domain for DrQA), then I imagine the results will be poor. DrQA isn't regularized to learn domain invariant features, and the datasets that it has been trained on do exhibit some strong domain biases (e.g. paraphrasing in SQuAD).

The bottom line however is that you can certainly customize the document retriever to operate on a different corpus (say the medical dataset). Using one of the pre-trained DrQA models you can also fine tune it with more in-domain distantly supervised data, if you have any QA pairs at hand.

Your results may vary, and I don't imagine it to be as transferable as ImageNet. But I can also see it working decently, depending on the target domain!
